build/module-builder/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:12-alpine as builder
+WORKDIR /botpress
+RUN apk add --no-cache git
+RUN git clone https://github.com/botpress/botpress.git
+
+FROM node:12-alpine
+# still need git, because of the gulp initStudio task!?
+RUN apk add --no-cache git
+WORKDIR /botpress
+# add botpress source dependencies from builder
+# install dependencies
+COPY --from=builder ./botpress/botpress/package.json ./botpress/botpress/yarn.lock ./
+RUN yarn install
+# add build config
+COPY --from=builder ./botpress/botpress/build ./build
+COPY --from=builder ./botpress/botpress/gulpfile.js ./
+# add botpress sdk
+COPY --from=builder ./botpress/botpress/src ./src
+# add source
+COPY --from=builder ./botpress/botpress/modules ./modules
+COPY --from=builder ./botpress/botpress/app.json ./app.json
+COPY --from=builder ./botpress/botpress/.eslintrc.js ./.eslintrc.js
+COPY --from=builder ./botpress/botpress/.tool-versions ./.tool-versions
+
+# TODO move into builder image if possible, expose build/module builder instead
+# fully build once, takes 10min+
+RUN yarn run gulp build
```

Comment:
I have a big issue here, the build takes way to long for me to invest more time debugging it. `undefined` is really not a helpful error message.
<img width="1223" alt="Screenshot 2021-04-20 at 17 21 46" src="https://user-images.githubusercontent.com/73602543/115422916-a3182b00-a1fd-11eb-8f45-1831f7cc5987.png">

Reply:
That is indeed a very uninformative message

Reply:
I will look into this

Reply:
Fixed in 3652b8b76bd0dbc5b0b5ad18d31ad3781d99aacc
Added more logging in 1bb5c89c1eab75707ebcea4633d0ccf6cce33f8a

Reply:
Thank you this helps a lot, the build is no longer broken now!